### PR TITLE
fix(components): remove drawer if no children

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

If you pass a drawer with no children you have a shadow on the right.

**What is the new behavior?**

No more drawer so no more shadow

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://166.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://166.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://166.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://166.talend.surge.sh)

File: packages/components/src/Drawer/Drawer.component.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ DrawerAnimation.propTypes = {
 };
 
 function Drawer({ className, style, children }) {
+	if (!children) {
```
Comment (acateland) [inline packages/components/src/Drawer/Drawer.component.js]: a snapshot test on this would be nice

Comment (jmfrancois) [inline packages/components/src/Drawer/Drawer.component.js]: thanks ! done

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://166.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://166.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://166.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://166.talend.surge.sh)

**Pull request merged**